Formula/aws-console.rb

```diff
@@ -0,0 +1,19 @@
+class AwsConsole < Formula
+  desc "Command-line to use AWS CLI credentials to launch the AWS console in a browser"
+  homepage "https://github.com/aws-cloudformation/rain"
+  url "https://github.com/aws-cloudformation/rain/archive/v1.1.1.tar.gz"
```

Comment:
Shouldn't it be called `rain`?

Reply:
The github repository `rain` contains commit library code and a number of command-line tools that use them. See the contents of https://github.com/aws-cloudformation/rain/tree/master/cmd

Reply:
Homebrew already has [`rain` formula](https://github.com/Homebrew/homebrew-core/blob/6fbf421b7c3983acfc9fd6ba00204db4d2913a34/Formula/rain.rb), maybe it would be better to add `aws-console` to it?

Reply:
aws-console is a standalone tool that doesn't require rain to be installed; it just shares some code with the rain binary.